examples/meta/generator/targets/cpp.json

```diff
@@ -45,7 +45,8 @@
         "ShortRealMatrix": "SGMatrix<float32_t>",
         "RealMatrix": "SGMatrix<float64_t>",
         "LongRealMatrix": "SGMatrix<floatmax_t>",
-        "ComplexMatrix": "SGMatrix<complex128_t>"
+        "ComplexMatrix": "SGMatrix<complex128_t>",
+        "DenseDistance": "CDenseDistance<float64_t>"
```

Comment:
I guess we would call this `RealDistance` to follow our old pattern of calling everything float64_t real